src/algorithms/highlevel/sbic.h

```diff
@@ -54,7 +56,7 @@ class SBic : public Algorithm {
     declareParameter("size2", "second pass window size [frames]", "[1,inf)", 200);
     declareParameter("inc2", "second pass increment [frames]", "[1,inf)", 20);
     declareParameter("cpw", "complexity penalty weight", "[0,inf)", 1.5);
-    declareParameter("minLength", "minimum length of a segment [frames]", "[1,inf)", 10);
+    declareParameter("minLength", "minimum length of a segment [frames]. EDIT LANDR 2016: Parameter disabled!", "[1,inf)", 10);
```

Comment:
Yes you could remove it entirely, as well as all the commented out code. This pull request and the change tree in github is enough in my opinion.